apps/api-extractor/src/generators/ApiModelGenerator.ts

```diff
@@ -934,6 +934,13 @@ export class ApiModelGenerator {
 
       parentApiItem.addMember(apiTypeAlias);
     }
+
+    if (
```

Comment:
I'm not 100% confident about this, but don't we only want to process child declarations if `apiTypeAlias === undefined`? (i.e. move this chunk into the `if` block above)